data-generation/src/test/scala/SnapshotDeltaSpec.scala

```diff
@@ -16,6 +17,17 @@ class SnapshotDeltaSpec extends AnyFlatSpec with SparkTestWrapper {
     assert(graphs.graphs.length == 2)
   }
 
+  // Can be ignored if "it" is swapped with "ignore"
+  it should "also have the correct amount when it is time based" in {
+    val updates = List(1, 1, 0, 0, 1, 1) // List of amount of updates for t_1, t_2 .. t_6
+    val logs = LogFactory().buildIrregularSequence(updates)
+    val logRDD = spark.sparkContext.parallelize(logs)
+    val snapshotModel = SnapshotDeltaObject.create(logRDD, Time(2))
+    assert(snapshotModel.logs.count() == 4)
+    assert(snapshotModel.graphs.length == 3)
+    compareGraph(snapshotModel.graphs(1), snapshotModel.graphs(1))
```

Comment:
```suggestion
    assertGraphSimilarity(snapshotModel.graphs(1), snapshotModel.graphs(2))
```
Tanken var å sammenligne to forskjellige regner jeg med

Reply:
ye, will fix

Reply:
Fikser du denne så burde ting være gooood! 👏